src/app/features/home/pages/image-upload/image-upload.component.ts

```diff
@@ -0,0 +1,36 @@
+
+import { Component } from '@angular/core';
+import { ImageUploadService } from '@data/services/image-upload.service';
+import { ImageCroppedEvent } from 'ngx-image-cropper';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-image-upload.component.ts',
+  templateUrl: './image-upload.component.html',
+  styleUrls: ['./image-upload.component.scss']
+})
+export class ImageUploadComponent {
```

Comment:
Eslint Fehler beheben indem du die Suggestion akzeptierst:

```suggestion
export class ImageUploadComponent implements OnDestroy {
```